fares-data-build-tool/create-data
# fix: service end date issue

# Description

-   If a service had no end date, it would use todays date instead. This fix stops that from happening.

# Testing instructions

-   Locally, none of the services have end dates, which makes it easy to test. You can also alter the code if you want to make them have end dates, to see how they look.

# Type of change
